doc/changes/changes_6.0.3.md

```diff
@@ -0,0 +1,13 @@
+# Virtual Schemas 6.0.3, released 2025-03-06
+
+Code name: Developer FAQ
+
+## Summary
+
+This release fixes add a developer FAQ to the root project of all virtual schemas that covers *language-independent* developer questions.
+
+We also removed language-specific content from this repository, since it is the portal to virtual schemas of all languages with the exceptions of links.
```

Comment:
```suggestion
We also removed language-specific content from this repository, since it is the portal to virtual schemas of all languages, see the links to the documentation of each of the specific virtual schemas.
```